components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -908,7 +913,14 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         let mut found = false;
         for (_, ids) in self.split_trace_mut() {
             if ids.remove(&region_id) {
+                let ids_len = ids.len();
                 found = true;
+                info!(
```

Comment:
maybe too many logs

Reply:
We have other logs in split anyway, this does not increase a lot logs. And in v2, split will not be as frequent as v1. So it's likely OK.